route53/src/listhosts.js

```diff
@@ -38,42 +41,45 @@ function compactResponse(data, type, match) {
 }
 
 async function _command(params, commandText, secrets = {}) {
-  const {
-    type = false,
```

Comment:
@ericswildens Do we need these? I couldn't find where they were used so I removed them.